src/components/Detail.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable react/prop-types */
```

Comment:
- [ ] Please remove this eslint disable line and include prop-types in the component.